emcc.py

```diff
@@ -379,6 +379,10 @@ def run():
   elif '-dumpmachine' in sys.argv:
     print(shared.get_llvm_target())
     exit(0)
+  
+  elif '-dumpversion' in sys.argv: # gcc's doc states "Print the compiler version [...] and don’t do anything else."
```

Comment:
It looks like python is tripping up over the apostrophe in this comment

Reply:
Sorry, my editor's autocorrection ... modern times, uff.

Will someone try without that apostrophe, or should I update the pull request?

Reply:
I think it's easiest to update the PR with another commit and let tests run here.